specification/migrate/resource-manager/Microsoft.Migrate/2017-11-11-preview/migrate.json

```diff
@@ -0,0 +1,2940 @@
+              "readOnly": true
+            }
+          }
+        },
+        "origin": {
+          "type": "string",
+          "description": "Origin of the operation.",
+          "readOnly": true
+        }
+      }
+    },
+    "DownloadUrl": {
+      "type": "object",
+      "description": "Download URL for assessment report.",
+      "properties": {
+        "assessmentReportUrl": {
+          "type": "string",
+          "description": "Hyperlink to download report.",
+          "readOnly": true
+        },
+        "expirationTime": {
+          "type": "string",
+          "description": "Expiry date of download url.",
+          "readOnly": true
+        }
+      }
+    },
+    "ProjectResultList": {
+      "type": "object",
+      "description": "List of projects.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of projects.",
+          "items": {
+            "$ref": "#/definitions/Project"
+          }
+        }
+      }
+    },
+    "MachineResultList": {
+      "type": "object",
+      "description": "List of machines.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of machines.",
+          "items": {
+            "$ref": "#/definitions/Machine"
+          }
+        }
+      }
+    },
+    "GroupResultList": {
+      "type": "object",
+      "description": "List of groups.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of groups.",
+          "items": {
+            "$ref": "#/definitions/Group"
+          }
+        }
+      }
+    },
+    "AssessedMachineResultList": {
+      "type": "object",
+      "description": "List of assessed machines.",
+      "properties": {
+        "value": {
+          "description": "List of assessed machines.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssessedMachine"
+          }
+        }
+      }
+    },
+    "AssessmentResultList": {
+      "type": "object",
+      "description": "List of assessments.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of assessments.",
+          "items": {
+            "$ref": "#/definitions/Assessment"
+          }
+        }
+      }
+    },
+    "OperationResultList": {
+      "type": "object",
+      "description": "List of API operations.",
+      "properties": {
+        "value": {
+          "type": "array",
+          "description": "List of operations.",
+          "items": {
+            "$ref": "#/definitions/Operation"
+          }
+        }
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourcegroups/{resourceGroupName}/providers/Microsoft.Migrate/projects": {
+      "get": {
+        "security": [
+          {
+            "azure_auth": [
+              "user_impersonation"
+            ]
+          }
+        ],
```

Comment:
This can be moved. It can be right at the start of the file

Reply:
BTW, this applies to all operations in this swagger

---
In reply to: [167111503](https://github.com/Azure/azure-rest-api-specs/pull/2446#discussion_r167111503) [](ancestors = 167111503)

Reply:
Done